api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -516,6 +523,37 @@ message HttpConnectionManager {
   bool strip_matching_host_port = 39;
 }
 
+// The configuration to customize local reply returned by Envoy.
+message LocalReplyConfig {
+  // Configuration of list of mappers which allows to filter and change local response.
+  // The mappers will be checked by the specified order until one is matched.
+  repeated ResponseMapper mappers = 1;
+
+  // The configuration to form response body from the :ref:`command operators <config_access_log_command_operators>`
+  // and to specify response content type as one of: plain/text or application/json.
```

Comment:
I think a casual reader will understand what is meant here on response body, but the content type is a bit confusing. Can you put an example inline in these docs or in the configuration docs?

Reply:
Will do